### ISSUE TITLE
Question about the vggPyramid

### ISSUE THREAD

**Issue by fatemehazimi990** [PruneTruong/GOCor#4]
Hello,
Thanks for sharing the code. Are the vggPyramid features pretrained and fixed during training the model (I noticed in the test_model they are fixed)? I was wondering if training from scratch would affect the convergence behavior of the gocor module. Thank you in advance for sharing your insight :)

Cheers,
Fatemeh

**Comment by PruneTruong** [comment]
Hi, yes in the GLUNet-GOCor introduced in the GOCor paper, the VGG-16 feature backbone is fixed. However, for PWC-Net-GOCor, the feature backbone is trained, which didn't affect the performance of the GOCor module. 
We also finetuned the VGG-16 in PDC-Net (which contains GOCor), without any issues. I only encountered some training instability when finetuning the backbone and training the label map y_theta on a certain dataset for PDC-Net. For this reason, in that version of GOCor, the label map is fixed. Hope that helps!
